packages/api-explorer/__tests__/JsonForm/index.test.jsx

```diff
@@ -83,23 +34,4 @@ describe('JSONForm ', () => {
     expect(props.onSubmit).toHaveBeenCalledTimes(1)
     expect(mockEvent.preventDefault).toHaveBeenCalledTimes(1)
   })
-
-  it('extend all json-editor editors', () => {
```

Comment:
Why this test is removed?

Reply:
This test was possible because json-editor uses a function (extend) to make the extension of a json-editor 's editor, and it tests "which function we overrides" in our customization.

Now we use a class, and we can't make this test in a simple way, and does not look so useful as a test.